contrib/scripts/qt5.py

```diff
@@ -0,0 +1,66 @@
+import os
+import re
+import functools
+
+import Qt
+
+def convert( fileName ) :
+
+	with open( fileName ) as f :
+		text = "".join( f.readlines() )
+		#print text
```

Comment:
The PR seems good to me.
My only comments are about this line (which is a commented out print that probably could be removed), and the documentation for this script.

I assume that the purpose of the script is to "Substitute QtWidgets for QtGui where needed" and "Replace deprecated `_qtImport` calls with `from Qt import` calls", which are comments found inside the code itself.

But I think it would be easier to get an idea of what the script is about if such comments were added at the top of the file as well.